File: OKR/OKR.Models/Entity/EvaluateTarget.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using OKR.Common.Models.Entity;

namespace OKR.Models.Entity
{
    public class EvaluateTarget : BaseEntity
    {
        [ForeignKey("DepartmentObjectives")]
        public Guid? DepartmentObjectivesId { get; set; }
        [ForeignKey("DepartmentObjectivesId")]
        public DepartmentObjectives? DepartmentObjectives { get; set; }

        [ForeignKey("UserObjectives")]
        public Guid? UserObjectivesId { get; set; }
        [ForeignKey("UserObjectivesId")]
        public UserObjectives? UserObjectives { get; set; }


        public string Content { get; set; }
    }
}

File: OKR.UI/src/Models/EvaluateTarget.ts
export class EvaluateTarget {
    departmentObjectivesId : string 
    userObjectivesId : string
    content : string 
    createOn : Date 
    createBy : string 
    id : string
    constructor(departmentObjectivesId: string, userObjectivesId: string, content: string, createOn: Date, createBy: string, id: string) {
        this.id = id
        this.departmentObjectivesId = departmentObjectivesId
        this.userObjectivesId = userObjectivesId
        this.content = content
        this.createOn = createOn
        this.createBy = createBy
    }
}